api-server/src/summarization/summarization.service.ts

```diff
@@ -0,0 +1,110 @@
+import { HttpException, Injectable } from '@nestjs/common';
+import Anthropic from '@anthropic-ai/sdk';
+
+@Injectable()
+export class SummarizationService {
+  private readonly client: Anthropic;
+
+  constructor() {
+    this.client = new Anthropic({ apiKey: process.env['ANTHROPIC_API_KEY'] });
+  }
```

Comment:
Can/should we make this fail gracefully if the API key isn't provided?

Maybe it's enough for the portal to gracefully handle errors if the key isn't there.

Reply:
yeah we should definitely log out some kind of warning and just skip this feature. I want this to all work with or without this enabled

Reply:
added a warning message which will get logged on server start similar to other messages 

![image](https://github.com/user-attachments/assets/5eb8b4bb-bbc4-457d-bc04-457e492bf821)

In this instance I'm updated so that after that initial message it will no longer warn so as not to be overly chatty and if you ask for tags it just returns a empty list

Reply:
e5000348df6ad16050cd2a0d1359946b518b6036